application/src/main/java/org/togetherjava/tjbot/commands/help/HelpThreadCommand.java

```diff
@@ -0,0 +1,212 @@
+package org.togetherjava.tjbot.commands.help;
+
+import com.github.benmanes.caffeine.cache.Cache;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.*;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.InteractionHook;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import net.dv8tion.jda.api.interactions.commands.build.SubcommandData;
+import net.dv8tion.jda.api.interactions.commands.build.SubcommandGroupData;
+import net.dv8tion.jda.api.requests.RestAction;
+import net.dv8tion.jda.api.requests.restaction.WebhookMessageUpdateAction;
+import org.togetherjava.tjbot.commands.SlashCommandAdapter;
+import org.togetherjava.tjbot.commands.SlashCommandVisibility;
+import org.togetherjava.tjbot.config.Config;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
+
+import static org.togetherjava.tjbot.commands.help.HelpSystemHelper.TITLE_COMPACT_LENGTH_MAX;
+import static org.togetherjava.tjbot.commands.help.HelpSystemHelper.TITLE_COMPACT_LENGTH_MIN;
+
+/**
+ * Implements the {@code /help-thread} command, which are contains special command for help threads
+ * only
+ */
+public final class HelpThreadCommand extends SlashCommandAdapter {
+    private static final int COOLDOWN_DURATION_VALUE = 30;
+    private static final ChronoUnit COOLDOWN_DURATION_UNIT = ChronoUnit.MINUTES;
+    private static final String CATEGORY = "category";
+    private static final String TITLE = "title";
+    private static final String CLOSE = "close";
+    private static final String BOOKMARK = "bookmark";
+
+    private static final Supplier<Cache<Long, Instant>> newCaffeine = () -> Caffeine.newBuilder()
+        .maximumSize(1_000)
+        .expireAfterAccess(COOLDOWN_DURATION_VALUE, TimeUnit.of(COOLDOWN_DURATION_UNIT))
+        .build();
+
+    private final HelpSystemHelper helper;
+    private final Cache<Long, Instant> helpThreadIdToLastCategoryChange = newCaffeine.get();
+    private final Cache<Long, Instant> helpThreadIdToLastTitleChange = newCaffeine.get();
+    private final Cache<Long, Instant> helpThreadIdToLastClose = newCaffeine.get();
+
+    /**
+     * Creates a new instance.
+     *
+     * @param config the config to use
+     * @param helper the helper to use
+     */
+    public HelpThreadCommand(Config config, HelpSystemHelper helper) {
+        super("help-thread", "Help thread specific commands", SlashCommandVisibility.GUILD);
+
```

Comment:
"Help thread specific commands" reads a bit odd. Maybe `"Commands for help threads, such as changing the category"`

Reply:
ig it doesn't really matter as the end user is not gonna see this anyways when using the commands